protos/arm_authorizer_server/arm_authorizer_server.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package mavsdk.rpc.arm_authorizer;
+
+import "mavsdk_options.proto";
+
+option java_package = "io.mavsdk.arm_authorizer";
+option java_outer_classname = "ArmAuthorizerProto";
+
+service ArmAuthorizerServerService {
+    // Subscribe to arm authorization request messages. Each request received should respond to using RespondArmAuthorization
+    rpc SubscribeArmAuthorization(SubscribeArmAuthorizationRequest) returns(stream SubscribeArmAuthorizationResponse) { option (mavsdk.options.async_type) = ASYNC; }
+
+    // Authorize arm for the specific time
+    rpc AcceptArmAuthorization(AcceptArmAuthorizationRequest) returns(AcceptArmAuthorizationResponse) { option (mavsdk.options.async_type) = SYNC; }
+
+    // Reject arm authorization request
+    rpc RejectArmAuthorization(RejectArmAuthorizationRequest) returns(RejectArmAuthorizationResponse) { option (mavsdk.options.async_type) = SYNC; }
+}
+
+// Messages for SubscribeArmAuthorization
+message SubscribeArmAuthorizationRequest {}
+message SubscribeArmAuthorizationResponse {
+    uint32 system_id = 1; // vehicle system id
+}
+
+// Messages for RespondArmAuthorization
+message AcceptArmAuthorizationRequest {
+    int32 valid_time = 1; // Time in seconds for which this authorization is valid
+}
+
+// Result type
+message AcceptArmAuthorizationResponse {
+    CommandAnswer command_answer = 1; // Result enum value
+}
+
+// Messages for RespondArmAuthorization
+message RejectArmAuthorizationRequest {
+    bool temporarily = 1; // True if the answer should be TEMPORARILY_REJECTED, false for DENIED
+    RejectionReason reason = 2; // Reason for the arm to be rejected
+    int32 extra_info = 3; // Extra information specific to the rejection reason (see https://mavlink.io/en/services/arm_authorization.html)
+}
+
+// Result type
+message RejectArmAuthorizationResponse {
+    CommandAnswer command_answer = 1; // Result enum value
+}
+
+enum CommandAnswer {
+    COMMAND_ANSWER_ACCEPTED = 0; // Command accepted
+    COMMAND_ANSWER_FAILED = 1; // Command failed
```

Comment:
I think we're meant to always have the return type `Result`. I'm surprised if it works without that.

Reply:
Hmm, It was working this way, but there were some problems with slightly different namings. I have now replaced the return type of `respond...` functions to be the same as in other plugins to keep it consistent